consumer/README.md

```diff
@@ -6,19 +6,34 @@ operated by a third party.
 ## Requirements
 
 * Access to a kcp workspace.
-* Path of the kcp workspace in which the service is deployed (e.g. `root:pipeline-service`).
-This path is provided by the service operators.
+* The following information from the service provider:
+  * Path of the kcp workspace in which the service is deployed (e.g. `root:pipeline-service`).
+  * Name of the APIExport (e.g. `kubernetes`).
 
-## How to bind to the service
+## Quick start guide
 
 * Login on your kcp workspace.
```

Comment:
nitpicking: Login to your kcp workspace.

Reply:
Done